src/modules/main/navbar/user-section/UserSection.scss

```diff
@@ -6,15 +6,20 @@
   .details {
     @include row;
     margin-right: 16px;
+    cursor: pointer;
 
     & > span {
       @include label;
-      margin-right: 16px;
+      margin-left: 16px;
     }
 
     .avatar {
       box-shadow: $shadowPrimary;
     }
+
+    > svg {
```

Comment:
& - add this - there was an issue with > syntax without & some time ago and right now we always using  & >